test/python/result/test_mitigators.py

```diff
@@ -491,5 +432,21 @@ def test_error_handling(self):
         )
 
 
+class TestLocalReadoutMitigation(QiskitTestCase):
+    """Tests specific to the local readout mitigator"""
+
+    def test_assignment_matrix(self):
+        """Tests that the local mitigator generates the full assignment matrix correctly"""
+        qubits = [7, 2, 3]
+        assignment_matrices = LocalReadoutMitigator(backend=FakeYorktown())._assignment_mats[0:3]
+        expected_assignment_matrix = np.kron(
+            np.kron(assignment_matrices[2], assignment_matrices[1]), assignment_matrices[0]
+        )
+        expected_mitigation_matrix = np.linalg.inv(expected_assignment_matrix)
```

Comment:
This is 100x better thanks - I can super clearly see what's going on, and it's obvious to me that this should be a valid mitigation strategy.